.travis.yml

```diff
@@ -1,7 +1,52 @@
 language: node_js
 node_js:
-  - "0.10"
+- '0.10'
 before_install:
-  - npm install -g npm@next
-  - npm install -g grunt-cli
-install: npm install
+- npm install -g npm@next
+- npm install -g grunt-cli
+install:
+- npm install
+- grunt build
+- grunt zip
+- grunt codedeploy
+deploy:
+  - provider: s3
+    access_key_id: AKIAIMTOX4BXGVFOB56Q
+    secret_access_key:
+      secure: LZToB5h0IoW4K21i2gYT4cdiWjzlb8rnz7pL8gL/y/gRX9UjWr2xHDtNgBbAi8nQcle0/z4NgwG2UzfaP4EmKN09P9WFyUtmIOtwO8cPMqg0L/W8pP6gWe/Ebfd42AILJ/niuklP79Hb9m71pNJfm+XhFbWXCEDAo3DYFa1uYEE=
+    skip_cleanup: true
+    bucket: hmda-pilot
+    local_dir: dist
+    upload-dir: milestone3
+    on:
+      branch: milestone3
```

Comment:
If you want to look over the options for conditional releases its documented here http://docs.travis-ci.com/user/deployment/